frontend/src/components/NavBar/PrintImage/image.tsx

```diff
@@ -556,6 +475,44 @@ function DownloadImage({ classes, open, handleClose }: DownloadImageProps) {
                       {titleText}
                     </div>
                   )}
+                  {footerTextSize > 0 && (
+                    <div
+                      ref={footerOverlayRef}
+                      className={classes.footerOverlay}
+                      style={{
+                        fontSize: `${footerTextSize}px`,
+                      }}
+                    >
+                      <div style={{ padding: '8px' }}>{footerText}</div>
+                    </div>
+                  )}
+                  {legendPosition !== 0 && (
+                    <div
+                      style={{
+                        position: 'absolute',
+                        zIndex: 2,
+                        top: titleOverlayRef?.current?.offsetHeight || 0,
+                        ...(legendPosition === 1
+                          ? { left: '8px' }
+                          : {
+                              right: `${
+                                (legendOverlayRef?.current?.offsetWidth || 0) *
```

Comment:
@echaidemenos there is a small bug when switching from "off" to "right", most likely due to the use of `legendOverlayRef?.current?.offsetWidth | 0` here no?

<img width="1725" alt="Screenshot 2024-04-20 at 9 53 24 AM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/3173206c-2cf2-4aba-a477-7737cc0b7178">